helm/prometheus-operator-app/values.yaml

```diff
@@ -322,7 +326,7 @@ alertmanager:
     ## Image of Alertmanager
     ##
     image:
-      repository: quay.io/giantswarm/alertmanager
+      repository: giantswarm/alertmanager
```

Comment:
Minor but we decided to rename repository to name.

https://github.com/giantswarm/fmt/blob/master/helm/image_templating.md

Reply:
Thanks, it made me notice some images I missed :)